### PR TITLE
docs(setup): :pencil: update readme files and repository URL

### PR DESCRIPTION
**Pull request by manh-vv**

### CODE REVIEW

**Review by manh-vv — COMMENTED** [review]
ready for review

File: packages/logger/README.md [hunk 1]
```diff
@@ -1,11 +1,37 @@
...
+
+declare global {
+  var logger: Logger;
+}
+
+// if consoleLogger is set to false, the function will create a null logger
+// which do nothing with log
+global.logger = createLogger({ consoleLogger: true });
+
+logger.debug('settup logger');
```
Comment (tiendq) [inline packages/logger/README.md:36]: Setup.

**Comment by tiendq** [comment]
LGTM.

**Review by manh-vv — COMMENTED** [review]
resolved discussions

**Pull request merged**